examples/basic_qa_pipeline.py

```diff
@@ -0,0 +1,76 @@
+import logging
+from pathlib import Path
+
+logging.basicConfig(format="%(levelname)s - %(name)s -  %(message)s", level=logging.WARNING)
+logging.getLogger("haystack").setLevel(logging.INFO)
+
+from haystack.document_stores import ElasticsearchDocumentStore
+from haystack.utils import clean_wiki_text, convert_files_to_docs, fetch_archive_from_http, print_answers, launch_es
+from haystack.nodes import FARMReader, BM25Retriever
+from haystack.nodes.file_classifier import FileTypeClassifier
+from haystack.nodes.preprocessor import PreProcessor
+from haystack.nodes.file_converter import TextConverter
+from haystack.pipelines import Pipeline
+from pprint import pprint
+
+
+def basic_qa_pipeline():
+    # launch and create DocumentStore
+    launch_es()
+
+    # Initialize a DocumentStore
+    document_store = ElasticsearchDocumentStore(host="localhost", username="", password="", index="document")
```

Comment:
I agree with @bilgeyucel, having all the params set explicitly, even to None, can be handy. Not a must, but a nice-to-have imho